evaluate/version.go

```diff
@@ -1,7 +1,7 @@
 package evaluate
 
 // Version holds the current version of the evaluation benchmark.
-var Version = "0.5.0"
+var Version = "0.6.2"
```

Comment:
Please open an issue that we automate this. Release -> update, automatic.

Reply:
https://github.com/symflower/eval-dev-quality/issues/344